docs/getting-started/typescript-composition-api.md

```diff
@@ -8,254 +8,154 @@ heading: Get Started
 position: 20
 ---
 
-# Get Started with Kendo UI for Vue Native Components with TypeScript and Composition API
+# Get Started with Kendo UI for Vue
 
-Recently, the [Composition API](https://vuejs.org/guide/introduction.html#composition-api) in Vue is getting more and more popular, over the classic [Options API](https://vuejs.org/guide/introduction.html#options-api). 
+This tutorial will help you develop a simple app that includes a native Vue Data Grid component. To achieve this, you will build a project using [Vite](https://vitejs.dev/) and the [Vue Composition API](https://vuejs.org/guide/introduction.html#composition-api) paired with TypeScript.
 
-In this article you will learn how to use **Kendo UI for Vue Native** components with `TypeScript` and `Composition API`. Below we are building a small app that includes a Grid, a DropDownList, a Window and a design theme.
 
-For the example discussed in the current article, we are using Vue 3. The Kendo UI for Vue Native components can be used with Composition API both in Vue 3 and Vue 2 context. More about the `Composition API`, you can learn on this [TypeScript Integration](slug:overview_typescript_integation) article. 
+>Curious about JavaScript or the Options API? This tutorial comes in several additional variants:
+>* [Kendo UI for Vue with TypeScript and the Options API](slug:getting_started_typescript_options_api)
+>* [Kendo UI for Vue with JavaScript and the Options API](slug:getting_started_javascript_options_api)
 
 > Vue 2 is currently in its end-of-support phase till Nov 2024. After our last major release for 2024, Vue 2 will no longer be supported in the new versions of the Kendo UI for Vue components. Please check our [Vue 2 End of Support article](slug:vue2-end-of-support) for more details.
 
-The current example uses the recommended by Vue [Vite build tool](https://vitejs.dev/) + [Vue Composition API](https://vuejs.org/guide/introduction.html#composition-api) + Typescript. If you need Getting started information that uses `Vue Options API` + Typescript, you can check [this link](slug:getting_started_typescript_options_api).
-
 > Prefer video tutorials? How about a free Telerik UI onboarding course? Check out the [Kendo UI for Vue with TypeScript](https://learn.telerik.com/learn/course/internal/view/elearning/45/kendo-ui-for-vue-with-typescript) training in [Telerik Virtual Classroom](https://learn.telerik.com/learn).
```

Comment:
This is the third callout in a row on a completely different subject. So many callouts in one place dilute the focus and might negatively impact the reader's experience. I would move this one elsewhere or delete it altogether.